src/feature/control/control.c

```diff
@@ -7121,41 +7122,36 @@ control_event_bootstrap(bootstrap_status_t status, int progress)
     }
   }
 
-  if (status > bootstrap_percent ||
-      (progress && progress > bootstrap_percent)) {
-    int loglevel = LOG_NOTICE;
-    bootstrap_status_to_string(status, &tag, &summary);
```

Comment:
By removing this, I believe `tag` and `summary` will be undefined if they hit the snprintf below.

Reply:
This is a minor mistake I made when rewriting history. 15c24d669f restores the call to `bootstrap_status_to_string()`, so the right thing happens if all the patches are applied. I can fix this one commit by editing the branch if you like.